_collections/_news/2025-06-24-year-in-review-2025.md

```diff
@@ -0,0 +1,231 @@
+welcoming a second iteration sometime next year. 
+
+In our AGM, we also passed an amendment adding the role of the Hackathons Officer, so if you enjoyed birmingHack and want 
+to help organise an even better follow-up to the debut event, or you're passionate about hackathons and want to get 
+yourself involved with birmingHack, this is the perfect role for you! Keep an eye out for the EGM that elects this role 
+early next year, around the time of October/November.
+
+Thank you to each and every committee member of CSS who helped make this event possible - and thank you as well to Adam,
+an external volunteer from hackSheffield who eagerly volunteered to help ensure the event went smoothly!
+
+If you want to see some of the other projects that were submitted, you can see them here: <https://birminghack.devpost.com/project-gallery>
+
+# Outside of Events - Discord and Minecraft
+We've also been working on things for you to do outside of term time - whether you need time to kill or want to reconnect
+with some university friends virtually, we've got you covered.
+
+First of all, as it was voted on in our AGM, we've allowed for members to select what colour they come up as on Discord!
+Whilst you're able to continue being the old-fashioned green you know and love, you can also select other colours that
+you may prefer. 
+
+![A screenshot of the member colour roles, including the original green, quiz victor pink, orange, purple, new green, yellow and red.](/assets/images/2025-end-of-year-newsletter/member-roles.png)
+
+If you haven't become a member on our Discord server yet, you can use `/makemember` to give yourself the Member role, or
+if you even aren't part of the Discord server, you can join our community over at <https://cssbham.com/discord>!
+
+However, if you have a friend who isn't a member but wants to get involved, then we've still got plenty going on. We've 
+added two new channels for you to enjoy, the first being the #activities channel, where you can start Discord activities 
+to play with other server members. We've currently got a Wordle streak of 20 days, but it's not just Wordle being played;
+you can do the daily sudoku, build a farm, race each other, fight each other in a suspiciously blocky FPS game, anything
+that comes up on Discord!
+
+![A screenshot of the activities channel, showing how everyone did in Wordle the previous day.](/assets/images/2025-end-of-year-newsletter/activities.png)
+
+And last but not least for our Discord server, we've added a new opt-in channel, the #pets🐶 channel! If you need a bit
+of extra serotonin to get you through the summer, server members have started posting their pets in this channel. You
+can access this channel through the #roles channel, clicking on the "Pets" reaction to get the respective role that shows
+the channel.
+
+![A screenshot of a portion of the pets channel, including two puppies, one tortoise, a dog and a cat.](/assets/images/2025-end-of-year-newsletter/pets.png)
```

Comment:
is it worth discussing small changes in Discord in such detail? (to me at least) it's not a very interesting topic

Reply:
The portion of the Discord was more targeted to those who interact less with it but could use these as a way to get involved over the summer - so it may be of less interest to us, but may be of interest to others